PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/BubbledGraphTest.java

```diff
@@ -0,0 +1,21 @@
+package nl.tudelft.pl2016gr2.model;
+
+public class BubbledGraphTest extends GraphTest {
+  
+  /**
+   * Setup, creates a new graph.
+   */
+  @Override
+  public GraphInterface getInstance() {
+    //ArrayList<Bubble> bubbles = new ArrayList<Bubble>();
+    Bubble b2 = new Bubble(2, 4);
+    Bubble b1 = new Bubble(5, 8);
+    //bubbles.add(b1);
+    //bubbles.add(b2);    
+    GraphInterface gra = new BubbledGraph();
```

Comment:
Local variable <code>gra</code> hides the public <code>gra</code> field of the parent class.